### PR TITLE
Add CHANGELOG.md, with entry for v0.4.0

### PR DESCRIPTION
**Pull request by timrogers**

### CODE REVIEW

File: rspec_junit_formatter.gemspec [hunk 1]
```diff
@@ -12,6 +12,10 @@ Gem::Specification.new do |s|
   s.required_ruby_version = ">= 2.0.0"
   s.required_rubygems_version = ">= 2.0.0"
 
+  s.metadata = {
+    'changelog_uri' => 'https://github.com/sj26/rspec_junit_formatter/blob/master/CHANGELOG.md',
```
Comment (sj26) [inline rspec_junit_formatter.gemspec]: Is this a standard?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# CHANGELOG
+
+## v0.4.0 (2018-05-26)
+
+* Add support for including STDOUT and STDERR from tests in the JUnit output (see ["Capturing output"](https://github.com/sj26/rspec_junit_formatter#capturing-output) in the readme for details)
+* When RSpec includes a diff in its output, strip out ANSI escape codes used to color it for shell display
+* Use [Appraisal](https://github.com/thoughtbot/appraisal) to test the gem against multiple versions of RSpec
+* Run gem tests against Ruby 2.5
```
Comment (sj26) [inline CHANGELOG.md]: Neat!

File: rspec_junit_formatter.gemspec (hunk 1 above)
Comment (sj26) [reply rspec_junit_formatter.gemspec]: Ah, I see:

https://guides.rubygems.org/specification-reference/#metadata

**Comment by sj26** [comment]
Thanks!

File: rspec_junit_formatter.gemspec (hunk 1 above)
Comment (timrogers) [reply rspec_junit_formatter.gemspec]: Yup! See the "Metadata" heading at https://guides.rubygems.org/specification-reference/.

**Pull request merged**